src/Microsoft.ComponentDetection.Detectors/linux/LinuxContainerDetector.cs

```diff
@@ -215,7 +215,10 @@ private async Task<int> GetBaseImageLayerCount(ContainerDetails scannedImageDeta
                 record.BaseImageLayerMessage = $"Base image annotations not found on image {image}, Results will not be mapped to base image layers";
                 this.Logger.LogInfo(record.BaseImageLayerMessage);
                 return 0;
-            } else if (scannedImageDetails.BaseImageRef == "scratch") {
+            }
+
+            if (scannedImageDetails.BaseImageRef == "scratch")
```

Comment:
was it intentional for the "else if" to turn into just an "if" statement

Reply:
Yes, it was a small optimization recommended by my IDE[^1]. I can revert if you want.

[^1]: https://www.jetbrains.com/help/resharper/RedundantIfElseBlock.html